api-samples/downloads/download_manager/README.md

```diff
@@ -0,0 +1,13 @@
+# chrome.downloads - Download Manager
+
+A sample that demonstrates how to use the [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) API.
+
+## Overview
+
+In this sample, multiple [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) APIs are comprehensively used to implement a simple download manager.
```

Comment:
```suggestion
This sample uses multiple [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) APIs to implement a simple download manager.
```

Reply:
'Comprehensive' = covering completely. Does this sample really use every aspect of `chrome.downloads`?